tests/imports.rs

```diff
@@ -82,6 +82,28 @@ fn single_quotes_import() {
     );
 }
 
+#[test]
+fn comma_seperated_import() {
+    let input = "@import 'firsta', 'seconda';\na {\n color: $a;\n}";
+    tempfile!("firsta", "$a: red;");
```

Comment:
It would be best if the file names included the test name as a kind of namespace. E.g. `tempfile!("comma_separated_import_a", "$a: red;");`